ustream-openssl.c

```diff
@@ -123,6 +123,8 @@ __ustream_ssl_context_new(bool server)
 # ifndef TLS_client_method
 #  define TLS_client_method SSLv23_client_method
 # endif
+#else
+	OPENSSL_init_ssl(0, NULL);
```

Comment:
This definitely merits close attention.  By looking at the code, the most probable cause of failure is that `SSL_CTX_new()` calls [`OPENSSL_init_ssl(OPENSSL_INIT_LOAD_SSL_STRINGS, NULL);`](https://github.com/openssl/openssl/blob/7fe57ffb29b9d6b65a7c88184cf1e010f74efc5f/ssl/ssl_lib.c#L3211) instead of `0`.

Reply:
Yeah, I didn't analyze why that happens yet, but without this multiple things broke. Like uhttpd, ddns and adblock.
I didn't try if this works when including the deprecated stuff. It's easily to verify with uclient-fetch and a https URL